stl/inc/yvals_core.h

```diff
@@ -855,6 +855,42 @@
 
 // next warning number: STL4027
 
+// P0619R4 remove deprecated features from C++20
+#ifndef _HAS_REMOVED_FEATURES_CXX20

```

Comment:
Naming: Would `_HAS_FEATURES_REMOVED_BY_CXX20` or `_HAS_FEATURES_REMOVED_IN_CXX20` be clearer?

Reply:
True, I went for `_HAS_FEATURES_REMOVED_IN_CXX20`